about.php

```diff
@@ -202,15 +202,15 @@
             <ul>
-              <li><a href="https://www.bl.uk/collection-items/engraving-of-the-screen-scene-from-sheridans-the-school-for-scandal">“Scene from <i>School for Scandal</i> being performed in Drury Lane Theatre, London,”</a> 1778, © The Trustees of the British Museum, CC-BY-NC-SA 4.0 license</li>
+              <li><a href="https://www.britishmuseum.org/collection/object/P_Ee-4-80">“Scene from <i>School for Scandal</i> being performed in Drury Lane Theatre, London,”</a> 1778, © The Trustees of the British Museum, CC-BY-NC-SA 4.0 license</li>
               <li><a href="https://collections.britishart.yale.edu/vufind/Record/3653228">William Blake, “<i>Beggar’s Opera</i>, Act III,”</a> c.1790, Yale Center for British Art, Paul Mellon Collection, Public Domain</li>
               <li><a href="https://collections.britishart.yale.edu/vufind/Record/1670672">Thomas Rowlandson, “An Audience Watching a Play at Drury Lane Theatre,”</a> c.1785, Yale Center for British Art, Paul Mellon Collection, Public Domain</li>
-              <li><a href="https://www.bl.uk/collection-items/playbills-for-performances-in-london-and-bristol-1774-77"> Playbill from Burney Collection, “‘Theatrical Register.’ A collection of playbills of London theatres, chiefly of Drury Lane, Covent Garden and the Haymarket,”</a>                1774-1777, British Library, Public Domain</li>
-              <li><a href="https://collections.folger.edu/detail/mr-garrick-delivering-his-ode-at-drury-lane-theatre-on-dedicating-a-building-%26-erecting-a-statue-to-shakespeare-/20b6c08e-1e0f-4167-bcf2-83fdf4f9bdff">John Lodge, “Mr. Garrick delivering his Ode at Drury Lane Theatre on dedicating a building & erecting a statue to Shakespeare,”</a>                Folger Shakespeare Library, CC-BY-SA 4.0 license</li>
+              <li><a href="https://www.digitens.org/en/media/341"> Playbill from Burney Collection, “‘Theatrical Register.’ A collection of playbills of London theatres, chiefly of Drury Lane, Covent Garden and the Haymarket,”</a>                1774-1777, British Library, Public Domain</li>
+              <li><a href="https://digitalcollections.folger.edu/img21902">John Lodge, “Mr. Garrick delivering his Ode at Drury Lane Theatre on dedicating a building & erecting a statue to Shakespeare,”</a>                Folger Shakespeare Library, CC-BY-SA 4.0 license</li>
               <li><a href="https://www.britishmuseum.org/research/collection_online/collection_object_details/collection_image_gallery.aspx?assetId=1613044433&objectId=3411178&partId=1">“Riot at Covent Garden Theatre, in 1763, in consequence of the Managers refusing to admit half-price in the Opera of <i>Artaxerxes</i>,”</a>                1763, © The Trustees of the British Museum, CC-BY-NC-SA 4.0 License</li>
               <li><a href="https://www.bl.uk/collection-items/portrait-of-aphra-behn-by-sir-peter-lely">Peter Lely, “Aphra Behn,”</a> 1670, © Yale Center for British Art, Public Domain</li>

```

Comment:
This link is also dead. Is this the correct source? https://collections.britishart.yale.edu/catalog/tms:51853